### ISSUE TITLE
comma-style and 'destructured' ES imports

### ISSUE THREAD

**Issue by wegry** [eslint/eslint#7551]
**What version are you using?**
3.9.1

**What did you do?**
Set comma-style to 'last' and ran it against an multiline ES import statement.

```
import {select
 ,timeParse
 ,scaleTime
 ,scaleLinear
 ,scaleOrdinal
 ,schemeCategory20c
 ,line
 ,curveBasis
 ,extent
 ,min
 ,min
 ,max
 ,max
 ,axisBottom
 ,axisLeft} from 'd3'
```

```
# .eslintrc.yaml
---
env:
  browser: true
  es6: true

parserOptions:
  ecmaVersion: 6
  sourceType: module

rules:
  # Auto fixed
  comma-style:
  - error
  - last
  semi:
  - error
  - never
  indent:
  - warn
  - 2
  - MemberExpression: 2
    SwitchCase: 1
    VariableDeclarator:
      var: 2
      let: 2
      const: 3
  no-extra-semi:
  - error
  no-unused-vars:
  - error
  - args: none
  no-redeclare:
  - error
  linebreak-style:
  - error
  - windows
  no-trailing-spaces:
  - error
  object-shorthand:
  - error
  no-var:
  - error
  quotes:
  - error
  - single
  - avoidEscape: true
    allowTemplateLiterals: true
  comma-spacing:
  - error
  - before: false
    after: true
  space-before-function-paren:
  - error
  space-before-blocks:
  - error
  space-infix-ops:
  - error
  eol-last:
  - error
  key-spacing:
  - error
  - mode: minimum
  no-multiple-empty-lines:
  - error
  arrow-parens:
  - error
  - as-needed
  arrow-spacing:
  - error

  # A little safety net
  no-dupe-keys:
  - error
  no-dupe-args:
  - error
  no-func-assign:
  - error
  no-undef:
  - error
  no-unreachable:
  - error
  valid-typeof:
  - error
  no-unsafe-negation:
  - error
  no-sparse-arrays:
  - error
  no-irregular-whitespace:
  - error
  use-isnan:
  - error
  no-empty:
  - error
  no-duplicate-imports:
  - error
  no-useless-rename:
  - error
  no-self-assign:
  - error
  no-useless-call:
  - error
  no-unmodified-loop-condition:
  - error
```

**What happened?**
`--fix` doesn't pick this up as I'd hoped.

**What did you expect to happen?**
Comma style ideally would pick up ES imports specified like this.

```
import {select,
  timeParse,
  scaleTime,
  scaleLinear,
  scaleOrdinal,
  schemeCategory20c,
  line,
  curveBasis,
  extent,
  min,
  min,
  max,
  max,
  axisBottom,
  axisLeft
} from 'd3'
```

**Comment by not-an-aardvark** [comment]
Thanks for the report.

However, I think this issue is a duplicate of https://github.com/eslint/eslint/issues/7470.

**Comment by wegry** [comment]
I'd certainly have no problem folding this issue into that one. I believe that [ES module named imports aren't in fact object destructing](http://stackoverflow.com/questions/39376073/destructuring-assignment-within-import-statements) which #7470 looks like it's about, but all I have to go off is a link to the spec. Maybe under the covers, eslint treats them the same.

**Comment by not-an-aardvark** [comment]
True; it looks like the issue itself doesn't mention named imports, but the [WIP fix](https://github.com/psocratic/eslint/commit/2a9a58187b1191d7079f1d9040b6f5714ecfa355#diff-8190ce567acc8a0b7a961df6bb3e3ba4R50) does account for them.

Since they're mostly the same thing, though, I think I'm going to leave a comment on the other issue to make sure that named imports stay on our radar for that fix, and then I'm going to close this issue so that we can address all of the comma-style problems in one issue.

**Comment by wegry** [comment]
Thanks for the info, and sounds good.
